assets/shaders/custom_material_chromatic_aberration.wgsl

```diff
@@ -1,14 +1,16 @@
 #import bevy_pbr::mesh_view_bindings
 
-[[group(1), binding(0)]]
+@group(1) @binding(0)
 var texture: texture_2d<f32>;
 
-[[group(1), binding(1)]]
+@group(1) @binding(1)
 var our_sampler: sampler;
 
-
-[[stage(fragment)]]
-fn fragment([[builtin(position)]] position: vec4<f32>) -> [[location(0)]] vec4<f32> {
+@fragment
+fn fragment(
+    @builtin(position) position: vec4<f32>,
+    #import bevy_sprite::mesh2d_vertex_output
```

Comment:
Is this import useful here? The diff doesn't seem to show that `bevy_sprite::mesh2d_vertex_output` data is used

Reply:
it is mandatory because of
> fragment inputs should now exactly match vertex outputs for locations, so I added exports for those to be able to reuse them https://github.com/gfx-rs/wgpu/pull/2704